pom.xml

```diff
@@ -59,6 +59,10 @@
     <jacoco.out.it.file>jacoco-it.exec</jacoco.out.it.file>
     <sonar.jacoco.reportPath>${jacoco.outputDir}/${jacoco.out.unit.file}</sonar.jacoco.reportPath>
     <sonar.jacoco.itReportPath>${jacoco.outputDir}/${jacoco.out.it.file}</sonar.jacoco.itReportPath>
+    <!-- default properties that can be altered on the command line -->
+    <fcrepo.test.context.path/>
```

Comment:
Unless there is a specific reason to include these two jacoco arg defaults, I would suggest leaving them out of the PR.
I am concerned that our Jenkins/Sonar reporting will become broken due to them.
- http://jenkins.fcrepo.org/
- http://sonar.fcrepo.org/